pkg/cmd/pulumi/deployment_settings_config.go

```diff
@@ -251,6 +260,18 @@ func initStackDeploymentCmd(
 		return err
 	}
 
+	oidcExplanationMsg := `
+Pulumi supports OpenID Connect (OIDC) integration across various services by
+leveraging signed, short-lived tokens and eliminating the necessity for hardcoded
+cloud provider credentials and facilitates the exchange of these tokens for
+short-term credentials.
```

Comment:
Should we keep this and the other messages on a single line and let the terminal do the wrapping?